site/docs/hive.md

```diff
@@ -84,7 +84,32 @@ You should now be able to issue Hive SQL `SELECT` queries using the above table
 SELECT * from table_b;
 ```
 
+#### Using Hadoop Catalog
+Iceberg tables created using `HadoopCatalog` are stored entirely in a directory in a filesytem like HDFS. 
+
+##### Create an Iceberg table
+The first step is to create an Iceberg table using the Spark/Java/Python API and `HadoopCatalog`. For the purposes of this documentation we will assume that the table is called `database_a.table_c` and that the table location is `hdfs://some_path/database_a/table_c`.
+
+##### Create a Hive table
+Now overlay a Hive table on top of this Iceberg table by issuing Hive DDL like so:
+```sql
+CREATE EXTERNAL TABLE table_a 
+STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' 
+LOCATION 'hdfs://some_bucket/some_path/database_a/table_c';
+```
+
+#### Query the Iceberg table via Hive
+TODO: why does below work if no config settings are set in Hive but fails if we add `set iceberg.mr.catalog=hadoop` like the code suggests we need to do?
+
+You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table. Both the Map Reduce and Tez query execution engines are supported.
+```sql
+SELECT * from table_c;
+```
+
 ### Features
 
 #### Predicate pushdown
 Pushdown of the Hive SQL `WHERE` clause has been implemented so that these filters are used at the Iceberg TableScan level as well as by the Parquet and ORC Readers.
+
+#### Column Projection
```

Comment:
This isn't related to the HadoopCatalog but since https://github.com/apache/iceberg/pull/1417 was just merged I thought I might as well just put it here.